woocommerce/woocommerce
# Fix styling of CES modal

### All Submissions:






### Changes proposed in this Pull Request:

CSS fix for the CES modal only caused in Wordpress version 6.1 and above

Needed for #35129  .






### How to test the changes in this Pull Request:



**Test the below steps using Wordpress 6.0.3 and 6.1 seperately.**
1. Load this branch, build it, and make sure you have the `new-product-management-experience` feature flag enabled (you can do so using the latest version of the Beta tester within the mono repo).
2. Create a couple products
3. Go to **Analytics > Products**
4. Select **Comparison** from the **Show** dropdown
5. Select a couple products and click **Compare**
6. A notice should show up that allows you to share feedback, click this.
7. Make sure the emoji's show up correctly in a horizontal format.
8. Note: that this notice will probably only show up once, once you click **Share feedback**, in order for it to show again you can delete the **woocommerce_ces_shown_for_actions** option.

Testing bonus points:
- 



### Other information:




### FOR PR REVIEWER ONLY:
